src/i18n/transifex_input.json

```diff
@@ -7,5 +7,13 @@
   "siteheader.user.menu.account.settings": "Account",
   "siteheader.user.menu.logout": "Logout",
   "siteheader.user.menu.login": "Login",
-  "siteheader.user.menu.register": "Sign Up"
+  "siteheader.user.menu.register": "Sign Up",
+  "profile.error.message.text": "An unexpected error occurred. Please click the button below to return to your profile and try again.",
```

Comment:
Should this be in here?

Reply:
Whoops.  Oversight on my part - I left the word "profile" in a few strings.  Will go fix and add to this PR...